docs/sources/reference/components/discovery/discovery.process.md

```diff
@@ -29,6 +29,7 @@ The following arguments are supported:
 | Name               | Type                | Description                                                                              | Default | Required |
 |--------------------|---------------------|------------------------------------------------------------------------------------------|---------|----------|
 | `join`             | `list(map(string))` | Join external targets to discovered processes targets based on `__container_id__` label. |         | no       |
+| `cgroup_id_regex`  | `string`            | Regex expression to retrieve cgroup ID of the process.                                   |         | no       |
```

Comment:
https://developers.google.com/style/word-list#regex

I think we should also document that the regular expression needs to have exactly one capture group which is what is used to extract the ID.

```suggestion
| `cgroup_id_regex`  | `string`            | Regular expression to retrieve cgroup ID of the process.                                   |         | no       |
```

Reply:
Yes, agree about documenting about capture groups. Modified the docs as suggested. Cheers!!